lib/facility_factory.rb

```diff
@@ -0,0 +1,35 @@
+class FacilityFactory
+
+  def initialize; end
```

Comment:
You don't even need to have this method. We aren't passing and args to our `FacilityFactory`.

Reply:
I think at the time it was more of a mental comfort thing. It was the first time I came across this situation and I'm a big fan of routine. I'm trying to break some habits that aren't useful.